app/calendar/page.tsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import "../styles.css";
 import "tailwindcss/tailwind.css";
+import BackgroundImage from "../components/BackgroundImage";
+import contactbackground from "../public/contactbackground.jpg";
 
 export default function Calendar() {
   return (
     <main>
-      <h2>Upcoming Events</h2>
+      <BackgroundImage imageSrc={contactbackground}>
```

Comment:
This is probably fine, but just check documentation to see if this is preferred way to do BG images, or if attaching the image to the `body` element in root `layout.tsx` is best.